wiki/Disambiguation/Score/id.md

```diff
@@ -0,0 +1,14 @@
+---
+tags:
+  - scoring
+  - skor
+---
+
+# Skor (disambiguasi)
+
+**Skor** dapat memiliki beberapa arti:
+
+- Pengiriman skor merupakan skor pemain pada sebuah [beatmap](/wiki/Beatmap)
```

Comment:
`- Submisi skor, yang merupakan proses pengiriman skor yang ditorehkan oleh seorang pemain pada sebuah [beatmap](/wiki/Beatmap)`

Reply:
`Submisi` isn't valid word

Reply:
there's `Submisi dan proses ranking beatmap` already in the main page tho ;~;

![image](https://user-images.githubusercontent.com/36564236/147138513-6258a691-3631-46f2-8b18-111a194eea2e.png)

hmm idk, i feel like (regardless of it being valid or not linguistically) the word `Submisi` is something that has been very widely used and understood nowadays. still this is up to you tho

Reply:
Ah if it used widely in the web we should add that